pom.xml

```diff
@@ -19,7 +19,7 @@
     <webappDirectory>
       ${project.build.directory}/${project.build.finalName}
     </webappDirectory>
-    <synapse.version>484.0</synapse.version>
+    <synapse.version>493.0</synapse.version>
```

Comment:
Bump to backend version that includes support for new EXEMPTION_ELIGIBLE permission (PLFM-8247 and PLFM-8248)